Dockerfile

```diff
@@ -1 +1,32 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/base_centos:7-20210630
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/centos/base-image:java8
```

Comment:
Why not use `ark_base_java8` here? Also, I think we can use Java 11 here, isn't it?

Reply:
Yes updated with `ark_base_java8`. We can use java11 also but as far as I know after going through the arkcase ce instance, solr was built on java8